JSDemos/Demos/TreeList/PopupEditing/React/App.tsx

```diff
@@ -33,6 +33,7 @@ const onInitNewRow = (e: TreeListTypes.InitNewRowEvent) => {
 const App = () => (
   <div id="tree-list-demo">
```

Comment:
Let's remove this container too (as you did in other demos)

Reply:
Let's do it in separate task for all DataGrid and React demos